tests/integration_tests/functional/test_signals.py

```diff
@@ -26,33 +26,21 @@ def test_sigbus_sigsegv(test_microvm_with_api, signum):
 
     test_microvm.basic_config()
 
-    # Configure logging.
-    log_fifo_path = os.path.join(test_microvm.path, 'log_fifo')
-    log_fifo = log_tools.Fifo(log_fifo_path)
-    response = test_microvm.logger.put(
-        log_path=test_microvm.create_jailed_resource(log_fifo_path),
-        level='Error',
-        show_level=False,
-        show_log_origin=False
-    )
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
-
     test_microvm.start()
     firecracker_pid = int(test_microvm.jailer_clone_pid)
 
     sleep(0.5)
     os.kill(firecracker_pid, signum)
 
     msg = 'Shutting down VM after intercepting signal {}'.format(signum)
-    log_fifo.flags = log_fifo.flags & ~os.O_NONBLOCK
-    lines = log_fifo.sequential_reader(5)
 
-    msg_found = False
-    for line in lines:
-        if msg in line:
-            msg_found = True
+    start = time()
```

Comment:
We could make the `_wait_for_message` function public and call it from here, right?

Reply:
Done.